Makefile

```diff
@@ -66,7 +66,7 @@ ifndef value
 	@echo To add a key, you need to provide its value in the "value" variable.
 	@exit 10
 endif
-	@$(PHP) ./cli/manipulate.translation.php -a add -k $(key) -v $(value)
+	@$(PHP) ./cli/manipulate.translation.php -a add -k $(key) -v "$(value)"
```

Comment:
Nitpick: it would have been better to have it in a separate commit to explain why you changed that ;)

Reply:
Done